### ISSUE TITLE
SheepShaver doesn't run in 64-bit mode on macOS 10.10+

### ISSUE THREAD

**Issue by vasi** [cebix/macemu#99]
Yields "Killed: 9". LLDB reports "Malformed Mach-o file". Maybe lowmem is doing something wrong?

**Comment by chrisballinger** [comment]
lowmem definitely corrupts the binary, and it would only run if I disabled
it during the build steps, but I guess that breaks other things that depend
on whatever lowmem is doing.

**Comment by asvitkine** [comment]
The 64-bit binaries work on older Mac OS X versions though. So it's
something that new versions of OS X reject.

**Comment by asvitkine** [comment]
So some questions I have is whether it's related to what OS lowmem runs on
and what binaries are input to it.

For example, it could choke on binaries compiled by clang - but might work
fine on binaries compiled by GCC and Xcode 3.2 on 10.6.

Or it could be that lowmem running on 10.12 doesn't work as intended.

**Comment by vasi** [comment]
I believe I compiled that version of SheepShaver on 10.6. Not 100% sure though.

**Comment by asvitkine** [comment]
Yep, I am able to reproduce those results now. It doesn't work on 10.11
either. Trying to debug what's wrong.

**Comment by asvitkine** [comment]
Here's the reason:

http://stackoverflow.com/a/26696361

It's a protection for 64-bit binaries that was added in 10.10. Basically,
it disallows page zero from having anything the NONE permission. Not sure
if there's anything we can do to workaround it, other than having
SheepShaver not require that page zero is mapped.

**Comment by asvitkine** [comment]
(The other workaround obviously is continuing using the 32-bit version
which still supports having page zero with VM_PROT_ALL permissions.)

**Comment by lequeu** [comment]
I have v 2.4.20140201 Working . with OS 9.0 on Mac OSX 10.12.3.. With no issues so far.  I did migrate the build from my 10.8.5 former machine
![screen shot 2018-03-09 at 4 04 29 pm](https://user-images.githubusercontent.com/37226753/37230007-f6da7282-23b3-11e8-8247-7567fe19990f.jpg)

**Comment by RonaldPR** [comment]
When you check Finder Info for the SheepShaver application you will see that the option "Open in 32-bit mode" is checked.

**Comment by lequeu** [comment]
Indeed. When checking the finder  "Open in 32bit mode" IS checked. 
![screen shot 2018-03-10 at 7 53 11 am](https://user-images.githubusercontent.com/37226753/37243008-8126e790-2440-11e8-8b70-70df65830a69.jpg)
